### PR TITLE
Direct Spark download url for CI as INFRA-12996

### PR DESCRIPTION
**Pull request by bzz**
`http://archive.apache.org/dist/spark/spark-${SPARK_VERSION}/${SPARK_ARCHIVE}.tgz` -> `http://d3kbcqa49mib13.cloudfront.net/${SPARK_ARCHIVE}.tgz`

### CODE REVIEW

**Comment by bzz** [comment]
Let's see if that helps CI and only after it's green - merge to #1689

**Comment by bzz** [comment]
CI Spar download pass now. Closing this in favour of #1709

File: .travis.yml [hunk 1]
```diff
@@ -42,9 +42,9 @@ matrix:
     - jdk: "oraclejdk7"
       env: SCALA_VER="2.11" PROFILE="-Prat" BUILD_FLAG="clean" TEST_FLAG="org.apache.rat:apache-rat-plugin:check -Dcheckstyle.skip=true" TEST_PROJECTS=""
```
Comment (Azizyahyaxx) [inline .travis.yml:43]: Connection to the server

**Review by Azizyahyaxx — APPROVED** [review]
https://www.terabox.com/wap/activity/recipient2?code=f_1020346790576129_4399349756757&pid=aGoldTask

**Pull request closed without merging**